torch/distributed/rpc.py

```diff
@@ -23,6 +24,15 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+def get_agent(agent):
+    return _agent
+
+
+def set_agent(agent):
+    global _agent
+    _agent = agent
+
+
```

Comment:
Are these intended to be a public API? If not, prefix with `_`.

Reply:
these are public.

Reply:
Can you add a docstring for them? E.g. is this just for overriding, when should you call these, etc.

Reply:
These are no longer used it seems?

Reply:
Removing.